examples/generate-and-verify-signature/generate_and_verify_signature_example.py

```diff
@@ -0,0 +1,62 @@
+import blocksec2go
+import hashlib
+import cryptography.exceptions as crypto_except
+
+def get_reader():
+  reader = None
+  reader_name = 'Identiv uTrust 3700 F'
+  while(reader == None):
+    try:
+      reader = blocksec2go.find_reader(reader_name)
+      print('Found the specified reader and a card!', end='\r')
+    except Exception as details:
+      if('No reader found' == str(details)):
+        print('No card reader found!     ', end='\r')
+      elif('No card on reader' == str(details)):
+        print('Found reader, but no card!', end='\r')
+      else:
+        print('ERROR: ' + str(details))
+        raise SystemExit
+  return reader
+
+def activate_card(reader):
+  try:
+    blocksec2go.select_app(reader)
+    print('Found the specified reader and a Blockchain Security 2Go card!')
+  except Exception as details:
+    print('ERROR: ' + str(details))
+    raise SystemExit
+
+def get_public_key(key_id):
+  try:
+    global_counter, counter, key = blocksec2go.get_key_info(reader, key_id)
+    print('Public key (hex, encoded according to SEC1): ' + key.hex())
+    return key
+  except Exception as details:
+    print('ERROR: ' + str(details))
+    raise SystemExit
+
+if('__main__' == __name__):
+  reader = get_reader()
+  activate_card(reader)
+
+  hash_object = hashlib.sha256(b'Hello World!')
```

Comment:
This will be changed in the next commit, I think we want to convert `Hello World!` string into bytes and then pass it to the hashlib library. It does not fundamentally change the example but it makes more sense